app/src/main/java/kr/ac/kaist/team888/mylittlefont/FontLoaderAdapter.java

```diff
@@ -58,7 +59,14 @@ public View getView(final int position, View convertView, ViewGroup parent) {
     fontDateView.setText(DATE_FORMAT.format(item.getDatetime()));
 
     // Make a locator for preview.
-    Locator locator = new Locator(PREVIEW_LETTER);
+    Locator locator;
+    if (preveiw) {
+      locator = new Locator(PREVIEW_LETTER[0]);
+    } else {
+      locator = new Locator(PREVIEW_LETTER[1]);
+    }
+    preveiw = !preveiw;
```

Comment:
I think this is not good for comparing several saved fonts at one time since preview letters of them in load dialog are different in two types. How about choosing preview letter randomly among multiple choices of letters? (with same preview letter in each item of loading list)

For multiple choices of letters, I'd like to include letters with flattening & arise feature dependencies.